From 735a2f1c8581dbb02d713daa8505023841d7cf5b Mon Sep 17 00:00:00 2001
From: Sujith Haridasan <Sujith_Haridasan@mentor.com>
Date: Tue, 15 Dec 2015 20:59:43 +0530
Subject: [PATCH] linux-xlnx: promote probe of ext4 ahead of ext2 & ext3

this patch provide probe of ext4 ahead of ext2 and ext3,
it is required to avoid the un-necessary probe of ext2
and ext3, when rootfs is of type ext4.

upstream-status: rebased as appropriate from awaiting upstream

patch reference: https://patchwork.ozlabs.org/patch/529498/

Signed-off-by: Sujith Haridasan <Sujith_Haridasan@mentor.com>
---
 fs/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/Makefile b/fs/Makefile
index 14bd7e8..23cd44b 100644
--- a/fs/Makefile
+++ b/fs/Makefile
@@ -62,11 +62,11 @@ obj-$(CONFIG_DLM)		+= dlm/
 # Do not add any filesystems before this line
 obj-$(CONFIG_FSCACHE)		+= fscache/
 obj-$(CONFIG_REISERFS_FS)	+= reiserfs/
+obj-$(CONFIG_EXT4_FS)		+= ext4/
 obj-$(CONFIG_EXT3_FS)		+= ext3/ # Before ext2 so root fs can be ext3
 obj-$(CONFIG_EXT2_FS)		+= ext2/
 # We place ext4 after ext2 so plain ext2 root fs's are mounted using ext2
 # unless explicitly requested by rootfstype
-obj-$(CONFIG_EXT4_FS)		+= ext4/
 obj-$(CONFIG_JBD)		+= jbd/
 obj-$(CONFIG_JBD2)		+= jbd2/
 obj-$(CONFIG_CRAMFS)		+= cramfs/
-- 
1.9.1

